### PR TITLE
Rename and Update BlueCava.com.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

**Comment by Bisaloo** [comment]
I get redirected to another domain but I don't see any indication that this domain is for sale. Could you point me to how you discovered this please?

**Comment by cschanaj** [comment]
You are right. I believe I have made a mistake typing www.bluecave.com when I work on this ruleset. Updated the title now. thanks.

File: src/chrome/content/rules/BlueCava.com.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	Non-functional hosts
+		Incomplete certificate chain:
+		- preferences.bluecava.com
```
Comment (Bisaloo) [inline src/chrome/content/rules/BlueCava.com.xml]: I get an expired cert

**Comment by cschanaj** [comment]
@Bisaloo Is there anything blocking PR?

**Comment by Bisaloo** [comment]
No sorry. Thanks for pinging me. I didn't merge it initially because I was waiting for travis to finish.

**Pull request merged**